$NetBSD$

Add NetBSD to list of operating systems that the OpenBSD provider is the
default for now. In fact, NetBSD's pkg_add provider can do a lot more
than OpenBSD's, but for now basic support should suffice.

--- lib/puppet/provider/package/openbsd.rb
+++ lib/puppet/provider/package/openbsd.rb
@@ -7,8 +7,8 @@ Puppet::Type.type(:package).provide :openbsd, :parent => Puppet::Provider::Packa
 
     commands :pkginfo => "pkg_info", :pkgadd => "pkg_add", :pkgdelete => "pkg_delete"
 
-    defaultfor :operatingsystem => :openbsd
-    confine :operatingsystem => :openbsd
+    defaultfor :operatingsystem => [:openbsd, :netbsd]
+    confine :operatingsystem => [:openbsd, :netbsd]
 
     def self.instances
         packages = []
